source/GarminSDView.mc

```diff
@@ -326,6 +329,16 @@ class SdDelegate extends Ui.BehaviorDelegate {
       )
     );
 
+    boolean = Storage.getValue(MENUITEM_O2SENSOR) ? true : false;
+    menu.addItem(
+      new Ui.ToggleMenuItem(
+        Ui.loadResource(Rez.Strings.O2_sensor_title),
+        Ui.loadResource(Rez.Strings.O2_sensor_desc),
+        MENUITEM_O2SENSOR,
+        true,
```

Comment:
I think this is the default value for the checkbox - it should be 'boolean' (badly named variable I know), so it is shown in the state from persistent storage?

Reply:
ok, I thougt the default value was defined here.